src/Wallabag/CoreBundle/Tools/Utils.php

```diff
@@ -4,6 +4,9 @@
 
 class Utils
 {
+
+    public const DEFAULT_WORDS_PER_MINUTE = 200;
```

Comment:
I would have extracted this refactor to a dedicated commit

Reply:
extracted this part in a dedicated commit